src/main/java/com/vladyslavpalamarchuk/trackmycoin/service/command/processor/RemoveMonitoringCommandProcessor.java

```diff
@@ -0,0 +1,113 @@
+package com.vladyslavpalamarchuk.trackmycoin.service.command.processor;
+
+import com.vladyslavpalamarchuk.trackmycoin.adaptors.persistence.MonitoringRepository;
+import com.vladyslavpalamarchuk.trackmycoin.adaptors.telegram.TelegramBotClient;
+import com.vladyslavpalamarchuk.trackmycoin.domain.Monitoring;
+import com.vladyslavpalamarchuk.trackmycoin.service.command.Command;
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import org.telegram.telegrambots.meta.api.objects.Update;
+
+@Component
+@RequiredArgsConstructor
+public class RemoveMonitoringCommandProcessor implements CommandProcessor {
+
+  private final TelegramBotClient telegramBotClient;
+  private final MonitoringRepository monitoringRepository;
+
+  private final String REMOVE_COMMAND = Command.REMOVE_MONITOR.getCommand();
+  private final int EXPECTED_INPUT_LENGTH = 2;
+  private final String TICKER_CURRENCY = "USDT";
+  private List<Monitoring> Monitorings;
+
+  @Override
+  public void process(Update update) {
+    Long chatId = getChatId(update);
+    String userMessage = update.getMessage().getText();
+
+    if (userMessage.equalsIgnoreCase(REMOVE_COMMAND)) {
+      telegramBotClient.sendMessage(
+          chatId, "Please enter the coin ticker and its current price\nfor example -> ETH 4000");
+      return;
+    }
+
+    processTickerAndPriceInput(update);
+  }
+
+  private void processTickerAndPriceInput(Update update) {
+    String[] inputParts = handleTickerInput(update, getChatId(update));
+    if (inputParts != null) {
+      remove(inputParts[0].toUpperCase() + TICKER_CURRENCY, inputParts[1], getChatId(update));
+    }
+  }
+
+  private boolean isCoinAvailable(String ticker, BigDecimal price) {
```

Comment:
isCoinAvailable is the same as you have for binance client. Please use different name for example: isMonitoringTickerExists